Assets/Scripts/Player/PlayerCamera.cs

```diff
@@ -27,10 +27,25 @@ public class PlayerCamera : ActorComponent
 	[SerializeField] Vector3 _focusOffset = new Vector3( 0f, 0f, 0.4f ); // How far offset is the camera it's intended position
 
 	// Zoom
-	[ReadOnly("ZoomDistance")] float _currentZoomDistance = 0f; // Current camera zoom
+	[ReadOnly("ZoomDistance")]
+	[SerializeField] float _currentZoomDistance = 0f; // Current camera zoom
+	[ReadOnly("Target Zoom Distance")]
 	[SerializeField] float _targetZoomDistance = 10f;
 
-	public DictionaryOfPhysicsStateTypeAndFloat _zoomDistanceStateMap = null;	// 10, 7
+	private float _zoomDistance = 10.0f;
+	public float zoomDistance
```

Comment:
What's the benefit of using a property here instead of making targetZoomDistance public since you're just assigning to it here anyway?